blueflood-http/src/main/java/com/rackspacecloud/blueflood/http/HttpResponder.java

```diff
@@ -34,6 +36,10 @@ public static void respond(ChannelHandlerContext ctx, HttpRequest req, HttpRespo
     }
 
     public static void respond(ChannelHandlerContext ctx, HttpRequest req, HttpResponse res) {
+
+        // set response headers
+        res.setHeader("Access-Control-Allow-Origin", Configuration.getInstance().getStringProperty(CoreConfig.CORS_ALLOWED_ORIGINS));
```

Comment:
If it's invalid, per spec, to send empty Access-Control-Allow-Origin, should we put this in an if statement? I.e: only sends back the header if the config has non empty configuration?